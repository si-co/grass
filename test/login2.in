# Missing login command
pass CrashOverride

# Wrong password
login Acidburn
pass whateverman

# login should reset after failed attempt
pass CrashOverride

# Correct login
login Acidburn
pass CrashOverride

# Can't do nested logins
login b

# Missing login command after login
pass b

exit
