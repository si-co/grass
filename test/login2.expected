# Missing login command
Error: call 'login' first

# Wrong password
Error : Wrong password

# login should reset after failed attempt
Error: call 'login' first

# correct login

# Can't do nested logins
Error : User is already logged in

# Missing login command after login
Error : User is already logged in
